### PR TITLE
Add at_app_flutter repo

### PR DESCRIPTION
**Pull request by xavierchanth**
**- What I did**

Add at_app_flutter, I needed to separate the flutter sdk dependencies from the at_app executable.
I've moved them to this at_app_flutter package.

**- How I did it**

Separate the necessary files from at_app and create a new package containing them.

**- How to verify it**

Verify by testing it with the at_app create command:

```sh
flutter pub global deactivate at_app
flutter pub global activate at_app
at_app create path/to/test/folder
```

**- Description for the changelog**
Add at_app_flutter package.

### CODE REVIEW

**Review by Intiserahmed — APPROVED** [review]

File: at_app_flutter/pubspec.lock [hunk 1]
```diff
@@ -0,0 +1,565 @@
+# Generated by pub
```
Comment (cpswan) [inline at_app_flutter/pubspec.lock]: I'm not sure that this file should be committed:

Per https://dart.dev/guides/libraries/private-files#pubspeclock

> For library packages, don’t commit the pubspec.lock file. Regenerating the pubspec.lock file lets you test your package against the latest compatible versions of its dependencies.

**Review by cpswan — CHANGES_REQUESTED** [review]
Some explanation of at_app_flutter/test/at_app_flutter_test.dart being empty would be useful

**Review by cpswan — APPROVED** [review]

File: at_app_flutter/pubspec.lock (hunk 1 above)
Comment (xavierchanth) [reply at_app_flutter/pubspec.lock]: Removed in the latest commit.

**Comment by xavierchanth** [comment]
> Some explanation of at_app_flutter/test/at_app_flutter_test.dart being empty would be useful

I'm having a hard time finding good tests to run for the library. I think an example app would be more useful here?

**Comment by cpswan** [comment]
@XavierChanth you could have a test (or set of tests) that exercise various aspects, and that would serve as an example. Or just don't have the empty file. Or put some //todo comments into the file.

**Comment by xavierchanth** [comment]
@cpswan Wrote unit tests for the two models in the package.

There is also the template lib/main.dart (skeleton app template) which uses the models.

Should I write a test for that file as well?

**Comment by cpswan** [comment]
You can add more later. Let's get what you have here merged @XavierChanth

**Pull request merged**